ch_5_AppDev_Intermediate/5_1_ChatBot.ipynb

```diff
@@ -105,10 +105,11 @@
     "\n",
     "> **Note**\n",
     ">\n",
-    "> * Currently, `load_in_low_bit` supports options `'sym_int4'`, `'asym_int4'`, `'sym_int5'`, `'asym_int5'` or `'sym_int8'`, in which 'sym' and 'asym' differentiate between symmetric and asymmetric quantization. Option `'nf4'` is also supported, referring to 4-bit NormalFloat.\n",
+    "> * Currently, `load_in_low_bit` supports options `'sym_int4'`, `'asym_int4'`, `'sym_int5'`, `'asym_int5'` or `'sym_int8'`, in which 'sym' and 'asym' differentiate between symmetric and asymmetric quantization. `'nf3'` and `'nf4'` stand for normalFloat quantizations . Floating point precision `'fp4'`, `'fp8'`, `'fp16'` and Option `'mixed_4bit'` are also supported.\n",
```

Comment:
may exclude nf3 for the time being. i.e. Keep `Option `'nf4'` is also supported, referring to 4-bit NormalFloat`.
Floating point precision -> floating point precision__s__
and mixed precisions including mixed_fp4 and mixed_fp8 are ...